packages/nuxt/src/app/composables/index.ts

```diff
@@ -1,36 +1,76 @@
-import type { UseHeadInput } from '@unhead/vue'
-import type { HeadAugmentations } from 'nuxt/schema'
+import type { UseHeadInput } from "@unhead/vue";
+import type { HeadAugmentations } from "nuxt/schema";
 
 /** @deprecated Use `UseHeadInput` from `@unhead/vue` instead. This may be removed in a future minor version. */
-export type MetaObject = UseHeadInput<HeadAugmentations>
+export type MetaObject = UseHeadInput<HeadAugmentations>;
 export {
   /** @deprecated Import `useHead` from `#imports` instead. This may be removed in a future minor version. */
   useHead,
   /** @deprecated Import `useSeoMeta` from `#imports` instead. This may be removed in a future minor version. */
   useSeoMeta,
   /** @deprecated Import `useServerSeoMeta` from `#imports` instead. This may be removed in a future minor version. */
-  useServerSeoMeta
-} from '@unhead/vue'
+  useServerSeoMeta,
+} from "@unhead/vue";
```

Comment:
Please remove auto-formatting changes (such as this one) from the PR ☺️

Reply:
Mentioned precisely that just above 😄

Reply:
pnpm lint:fix doesn't fix it so i'm reverting the changes and working on that again. this time with prettier turned off

Reply:
should be good now